client/src/components/DateTime/style.module.scss

```diff
@@ -0,0 +1,12 @@
+
+.text{
+
+    color: #7d7d7d; 
+    font-family: 'Ropa Sans';
```

Comment:
Can you add `sans-serif` as a fall-back font, change font-size to `1.5rem`, and remove the leading & trailing blank lines